torchagents/utilities/NNBuilder.py

```diff
@@ -0,0 +1,30 @@
+import torch
+from typing import List
+
+def LinearSequentialBuilder(
+        input_size: int,
+        output_size: int,
+        hidden_layers: List[int],
+        output_function: torch.nn = None,
+):
```

Comment:
Is there a reason why "):" is in a newline?

Reply:
No reason